src/com/google/enterprise/adaptor/fs/FsAdaptor.java

```diff
@@ -700,7 +715,9 @@ private void validateShare(Path sharePath) throws IOException {
     // Verify that the adaptor has permission to read the Acl and share Acl.
     try {
       readShareAcls(sharePath);
-      delegate.getAclViews(sharePath);
+      if (!delegate.isDfsNamespace(sharePath) || allowFilesInDfsNamespaces) {
```

Comment:
Does something go wrong when getAclViews() is called on DFS namespace? Why purposefully avoid?